workflow.txt

```diff
@@ -1,10 +1,9 @@
+fork on github
 git clone
 git checkout -b my-solution
-git commit -m ""
-git fetch --all
 git add .
-git pull upstream master
-git merge my-solution
-fork on github
+git commit -m ""
 git checkout master
+git merge my-solution
```

Comment:
👍  marking this as correct, but don't think you need the merge or pull locally, since this was asking about making a pull request on github